pincer/objects/guild/guild.py

```diff
@@ -1295,8 +1298,8 @@ async def create_emoji(
         self,
         *,
         name: str,
-        image: str,
-        roles: List[Snowflake],
+        image: File,
+        roles: List[Snowflake] = [],
```

Comment:
Was this intended?

Reply:
Yeah, having to manually specify an empty arrow for no permissions didn't make sense imo.